protocol/app/app.go

```diff
@@ -552,7 +576,9 @@ func New(
 
 	// Create static IBC router, add transfer route, then set and seal it
 	ibcRouter := ibcporttypes.NewRouter()
-	ibcRouter.AddRoute(ibctransfertypes.ModuleName, transferIBCModule)
+	ibcRouter.AddRoute(ibctransfertypes.ModuleName, transferIBCModule).
```

Comment:
Note: ordering of `AddRoute` does not matter here

Reply:
1. Added comment
2. I actually prefer the linked pattern since it's slightly less verbose, and is also used in official code [example](https://ibc.cosmos.network/main/apps/interchain-accounts/integration#example-integration)
I did change this to 
```
	ibcRouter.
		AddRoute(ibctransfertypes.ModuleName, transferIBCModule).
		AddRoute(icahosttypes.SubModuleName, icaHostIBCModule)
```